### PR TITLE
JENA-2089: Documentation for Data+RDFS

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: source/documentation/rdfs/__index.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+derived using RDFS. It supports `rdfs:subClassOf`, `rdfs:subPropertyOf`, `rdfs:domain` and `rdfs:range`. It
+does not provide RDF axioms. The RDFS vocabulary is not included in the data.
+
+It does support use with RDF datasets, where each graph in the dataset has the
+same RDFS vocabulary applied to it.
+
+This is not a replacement for the [Jena RDFS Reasoner support](/inference/)
+which covers full RDFS inference.
+
+The data is updateable, and graps can be added and removed from the dataset.
```
Comment (kinow) [inline source/documentation/rdfs/__index.md:16]: s/graps/graphs

File: source/documentation/rdfs/__index.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+The vocabulary can not be changed during th elifetime of the RDFS dataset.
```
Comment (kinow) [inline source/documentation/rdfs/__index.md:17]: s/th elifetime/the lifetime

File: source/documentation/rdfs/__index.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+### API: `RDFSFactory`
+
+The API provides operation to build RDF-enabled datasets from data storage and vocabularies:
+    
+Example:
+```
+    DatasetGraph data = ...
+    // Load the vocabulary
+    Graph vocab = RDFDataMgr.loadGraph("vocabulary.ttl");
+    // Create a DatsetGraph with RDFS
```
Comment (kinow) [inline source/documentation/rdfs/__index.md:28]: s/DatsetGraph /DatasetGraph

File: source/documentation/rdfs/__index.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    DatasetGraph dsg = datasetRDFS(DatasetGraph data, Graph vocab );
+    // (Optional) Present as a Dataset.
+    Dataset dataset = DatasetFactory.wrap(dsg);
+```
+
+The vocabulary is processed to produce datastructure needed for processing he
```
Comment (kinow) [inline source/documentation/rdfs/__index.md:34]: s/he/the

File: source/documentation/rdfs/__index.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+data eficiently at run time. This is the `SetupRDFS` class that can be created
+and shared; it is thread-safe.
+
+```
+    SetupRDFS setup = setupRDFS(vocab);
+```
+
+### Assembler: RDFS Dataset
+
+Datasets with RDFS can built with an assembler:
```
Comment (kinow) [inline source/documentation/rdfs/__index.md:44]: be built?

File: source/documentation/rdfs/__index.md (hunk 2 above)
Comment (afs) [reply source/documentation/rdfs/__index.md:17]: "elife"

**Comment by afs** [comment]
Fixes applied - thanks!

**Review by kinow — APPROVED** [review]

**Comment by namedgraph** [comment]
@afs is this available in Fuseki?

**Comment by afs** [comment]
See the documentation!

https://jena.apache.org/documentation/rdfs/#use-with-fuseki

and the mentioned ticket:
https://issues.apache.org/jira/browse/JENA-2089

so not in the v4.0.0 release.

**Pull request merged**